src/screen.cpp

```diff
@@ -0,0 +1,58 @@
+#include "../include/screen.h"
+
+// Constructor
+Screen::Screen(int width, int height, const std::string& title){
+  m_width = width;
+  m_height = height;
+  m_title = title;
+}
+
+Screen::~Screen(){
+  SDL_GL_DeleteContext(m_glContext);
+  SDL_DestroyWindow(m_window);
+  SDL_Quit();
+}
+
+// Screen initializer
+void Screen::Init(){
```

Comment:
Is there a reason not to do this in the constructor? Will the destructor cause trouble if the `Screen` object is destroyed before `Init()` is called? Keep [RAII](https://en.wikipedia.org/wiki/Resource_Acquisition_Is_Initialization) in mind.